src/guide/jbs-jdk-bug-system.md

```diff
@@ -5,52 +5,296 @@
+    * To help with setting the right priority, consider things like how the bug impacts the product and our testing, how likely is it that the bug is triggered, how difficult is it to work around, and whether it's a regression, since that may break existing applications. Regressions are often higher priority than long-standing bugs and may block a release if not fixed. An example of a P1 would be an issue that is blocking a build or a release, whereas a P5 would be a minor typo in a code comment - most bugs will be P3 or P4.
 * In the [Description]{.jbs-field}, always include (if possible):
-  * full name of the failing tests
-  * error messages
-  * assert messages
-  * stack trace
-  * command line information
-  * relevant information from the logs
-* If the failure isn't reproducible with an existing OpenJDK test, attach a reproducer if possible.
-* Only set [CPU]{.jbs-field} and/or [OS]{.jbs-field} fields if the bug **ONLY** happens on that particular platform.
+    * error messages
+    * assert messages
+    * stack trace
+    * command line information
+    * relevant information from the logs
+    * full name of any failing tests
+* Avoid including in the description or comments:
+    * personal information
+    * passwords, logins, machine names
+    * logs which may include sensitive data
+* If the failure isn't reproducible with an existing OpenJDK test, attach a reproducer if possible, while in a number of cases it isn't possible, having a test case will decrease the time required to resolve the issue.
+* Only set [CPU]{.jbs-field} and/or [OS]{.jbs-field} fields if the bug **ONLY** happens on that particular platform or set of platforms.
+* Including the java -fullversion is encouraged for bugs in the JVM, hangs, network issues where the exact version can be critical to determine what fixes may be missing from an older version.
 * Always file separate bugs for different issues.
-  * If two crashes looks related but not similar enough to for sure be the same, it's easier to close a bug as a duplicate than it is to extract the relevant info from a bug to create a new one later.
+    * If two crashes looks related, but not similar enough to be sure they are the same, it's easier to later close a bug as a duplicate than it is to separate out one bug into two.
+
+Things to keep in mind when requesting an improvement:
+
+* Unlike reporting a problem, when it comes to improvements what constitutes a reasonable request can take discussion, and in general it is encouraged that the mailing list for the area is used to suggest an improvement before filing.
+* Enhancements to The Java Language Specification and The JVM Specification are managed through the [Java Community Process](http://jcp.org/)
 
 To find out which component to use for different bugs, consult the [directory to area mapping](#directory-to-area-mapping).
 
+### Implementing a Large Change
```

Comment:
Hmm. I don't think I entirely agree with the guidance here.

I've often used subtasks of a bug/enhancement to break the work into logical pieces (e.g. porting FDLIBM JDK-8171407) or partitioning work along review/maintenance domains (e.g. JDK-8231641: Suppress warnings on non-serializable non-transient instance fields in JDK libraries (umbrella)).

I think such usages work with the features of JBS in terms of allowing an easy estimate of how much of the umbrella bug is left to do and looser connections between the cloud of issues obscures the intent of what is going on.

Reply:
I believe that the examples you give are covered in the last sentence of that section, you examples are however good examples of where sub-tasks can be used to effect and so have included them as examples in that section